content/docs/guides/webassembly/resources.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Additional Resources"
+weight: 4
+description: |
+  Additional Resources for using TinyGo with WASM/WASI
+---
+
+Here are a few resources for helping learn more about using TinyGo with WebAssembly.
+
+**Wasm By Example**
+https://wasmbyexample.dev/
+
+**Are We Wasm Yet ? - Part 1**
+https://elewis.dev/are-we-wasm-yet-part-1
+
+**Are We Wasm Yet ? - Part 2**
+https://elewis.dev/are-we-wasm-yet-part-2
+
+**Writing a WebAssembly Service in TinyGo for Wagi and Spin**
+https://www.fermyon.com/blog/tinygo-webassembly-favicon-server
+
+**WasmEdge Runtime - Go**
+https://wasmedge.org/book/en/write_wasm/go.html
+
+**WASI and Node.js**
+https://k33g.hashnode.dev/series/wasi-nodejs
+
+**Wazero, first steps**
+https://k33g.hashnode.dev/series/wazero-first-steps
```

Comment:
Some things I noticed:

```go
// readBufferFromMemory returns a buffer from WebAssembly
func readBufferFromMemory(bufferPosition *uint32, length uint32) []byte {
    subjectBuffer := make([]byte, length)
    pointer := uintptr(unsafe.Pointer(bufferPosition))
    for i := 0; i < int(length); i++ {
        s := *(*int32)(unsafe.Pointer(pointer + uintptr(i)))
        subjectBuffer[i] = byte(s)
    }
    return subjectBuffer
}
```

This is one example where `unsafe.Slice` would be perfect (although I'm not sure about the uint32 vs byte conversion, or what is supposed to be happening there):

```go
// readBufferFromMemory returns a buffer from WebAssembly
func readBufferFromMemory(bufferPosition *byte, wordLength uint32) []byte {
    length := wordLength * 4
    subjectBuffer := make([]byte, length)
    copy(subjectBuffer, unsafe.Slice(bufferPosition, length))
    return subjectBuffer
}
```